Various corrections to Ubuntu Core image creation guide (#22)
canonical/tf-lite-examples-snap: ubuntu-core-image/README.md

Wait until the computer info shows and then go to the **Snaps** tab.
Here you can click on **Install snaps**.
Enter snap name `ubuntu-frame` and `24/stable` for _tracking channnel_.
Click **+** and also add `wpe-webkit-mir-kiosk` and `22/candidate`.
Click **install**.

![install snap](../media/landscape-install.png)

This is equivalent to running the following on the device:
```
sudo snap install ubuntu-frame --channel=24/stable
sudo snap install wpe-webkit-mir-kiosk --channel=22/candidate

> ```sudo

Reviewer: Remove rogue `sudo`

The installation command will take some time to be delivered to the device, and then executed.
The result will be shown on Landscape.

If the installation was successful, you should see the annotated camera feed being displayed on the local display connected to the Raspberry Pi.

![local monitor](../media/loca-monitor-frame.jpg)

> The URL that is served by the `wpe-webkit-mir-kiosk` is set to the `tf-ubuntu-frame-example`'s web server by default, in the [gadget.yaml](additions-to-gadget.yaml).

Reviewer: I'm not sure `served` is the correct word to use here, as the URL is served by the python script. The kiosk browser visits that URL, and displays it on the screen.

This URL can be changed by setting snap options via a script on Landscape.
An example script to do this can be found [here](https://github.com/canonical/landscape-scripts/blob/main/core/snaps/update-snap-config.py).

## Limitations
- After a reboot, the Web Kiosk fails to show the web page in case the web server hasn't fully started. In this case, it is necessary to restart the `wpe-webkit-mir-kiosk` snap.

> - The USB camera should be connected before boot. Otherwise, after connecting the tf-lite application and Mir Kiosk should be restarted in the mentioned order.

Reviewer: ```suggestion
- The USB camera should be connected before boot. If the camera is connected after boot, `tf-ubuntu-frame-example` should be restarted after connecting the camera, followed by restarting `wpe-webkit-mir-kiosk`.
```
